blocks/reader-revenue-manager/block-editor-plugin/editor-styles.scss

```diff
@@ -16,7 +16,14 @@
  * limitations under the License.
  */
 
-/*
- * Block editor styles.
- */
-@import "components/reader-revenue-manager/googlesitekit-rrm-block-editor";
+.editor-sidebar {
+	.googlesitekit-rrm-settings-panel {
+
+		.googlesitekit-rrm-panel__select-control {
+			label {
+				text-transform: none;
+				white-space: break-spaces;
```

Comment:
Let's go with `normal`, there's no need to preserve multiple spaces in the way `break-spaces` does. I'll apply this change myself to save the round trip.